timescaledb/overview/release-notes/index.md

```diff
@@ -34,23 +31,34 @@ while also empowering users with more control and flexibility to customize
 behaviors to suit your needs.  
 Some of these API updates are **breaking changes**.
 
-### What's new in TimescaleDB 2.3.1:
+### What's new in TimescaleDB 2.4:
+
+This release adds new experimental features since the 2.3.1 release.
 
-This maintenance release adds support for downgrading Timescale DB 2.3.1 to 2.3.
-We intend to keep expanding this functionality as we release new versions to allow users to downgrade one version at a time.
+The experimental features in this release are:
+* APIs for chunk manipulation across data nodes in a distributed
+hypertable setup. This includes the ability to add a data node and move
+chunks to the new data node for cluster rebalancing.
+* The `time_bucket_ng` function, a newer version of `time_bucket`. This function
+supports years, months, days, hours, minutes, and seconds.
+ 
+We’re committed to developing these experiments, giving the community a chance to 
+provide early feedback and influence the direction of TimescaleDB’s development.
+ We’ll travel faster with your input! 
+	
+Please create your feedback as a GitHub issue (using the experimental-schema label), describe what you found, and tell us the steps or share the code snip to recreate it.
 
 Several bugs fixed, see the release notes for more details.
 
 **PostgreSQL 11 deprecation announcement**
 
+PostgreSQL 11 deprecation announcement
 Timescale is working hard on our next exciting features. 
-To make that possible, we require functionality 
-that is available in Postgres 12 and above. 
-For this reason, TimescaleDB 2.3.1 is the final version that supports PostgreSQL 11.  
-TimescaleDB 2.4 will require PostgreSQL 12 or 13.
+To make that possible, we require functionality that is available in Postgres 12 and above. 
+For this reason, from TimescaleDB 2.4 onwards PostgreSQL 12 or 13 are required.
```

Comment:
For this reason, we are removing support for PostgreSQL 11 starting in the TimescaleDB 2.4 release.  For
TimescaleDB 2.4 and onwards, PostgreSQL 12 or 13 are required.